golem-templates/templates/python/python-app-component-example-gitpulse-agent/components-python/component-name/golem.yaml

```diff
@@ -0,0 +1,74 @@
+# Schema for IDEA:
```

Comment:
we have replaces for these now:
https://github.com/golemcloud/golem-cli/blob/2ec69d35caa8267fde2a8927ec888982e349208b/golem-templates/templates/python/python-app-component-default/components-python/component-name/golem.yaml#L1-L7